content/en/docs/tasks/administer-cluster/subpath-env.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using subPath with expanded environment variables 
+content_type: task
+---
+{{< feature-state for_k8s_version="v1.17" state="stable" >}}
+
+<!-- overview -->
+Use the `subPathExpr` field to construct `subPath` directory names from
+downward API environment variables. The `subPath` and `subPathExpr` properties are mutually exclusive.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
```

Comment:
This jumps into a manifest with little context. What task is the reader learning to perform?

Reply:
I have revised this section with a new, friendlier, intro.